DEVELOPERS.md

```diff
@@ -24,28 +24,28 @@ machine:
   We recommend using [nvm][nvm] (or [nvm-windows][nvm-windows])
   to manage and install Node.js, which makes it easy to change the version of Node.js per project.
 
-### Forking Cicero on Github
+### Forking Concerto on Github
 
-To contribute code to Cicero, you must have a GitHub account so you can push code to your own
-fork of Cicero and open Pull Requests in the [GitHub Repository][github].
+To contribute code to Concerto, you must have a GitHub account so you can push code to your own
+fork of Concerto and open Pull Requests in the [GitHub Repository][github].
 
 To create a Github account, follow the instructions [here][github-signup].
 Afterwards, go ahead and [fork][github-forking] the
-[main Cicero repository][github].
+[main Concerto repository][github].
 
-### Building Cicero
+### Building Concerto
 
-To build Cicero, you clone the source code repository and use lerna to build:
+To build Concerto, you clone the source code repository and use lerna to build:
 
 ```shell
 # Clone your Github repository:
-git clone https://github.com/<github username>/cicero.git
+git clone https://github.com/<github username>/Concerto.git
 
-# Go to the Cicero directory:
-cd cicero
+# Go to the Concerto directory:
+cd Concerto
 
-# Add the main Cicero repository as an upstream remote to your repository:
-git remote add upstream "https://github.com/acccordproject/cicero.git"
+# Add the main Concerto repository as an upstream remote to your repository:
+git remote add upstream "https://github.com/accordproject/Concerto.git"
```

Comment:
This should be in lowercase (repository name).

Reply:
changed